packages/components/addon/components/hds/alert/index.hbs

```diff
@@ -17,10 +17,10 @@
   {{/if}}
 
   <div class="hds-alert__content">
-    <div class="hds-alert__text">
+    <Hds::Alert::Text @type={{@type}}>
```

Comment:
No strong opinions, just a consideration: _in theory_ you could have also used the handlebars helpers this way:
```
<div class="hds-alert__text {{if (eq @type 'compact') 'hds-typography-body-100' 'hds-typography-body-200'}}">
```
I see pros and cons in both, so whatever works better for you, I'm OK with it.

Reply:
(+1 to template conditionals)

Reply:
I probably would have done something like that but it doesn't seem to be our preferred pattern for handling conditional class names so that's why I ended up creating a new child component.

Reply:
I can redo it. I just didn't think that was the way we preferred.

Reply:
I refactored to remove the child component and just do conditional class names in the main template. Thanks

Reply:
> I can redo it. I just didn't think that was the way we preferred.

It is not. But in this very specific case, considering the pros and cons (having to create a component only for this) using inline conditionals is probably an acceptable tradeoff.